synapse/handlers/sync.py

```diff
@@ -213,6 +215,7 @@ class SyncResult:
     to_device = attr.ib(type=List[JsonDict])
     device_lists = attr.ib(type=DeviceLists)
     device_one_time_keys_count = attr.ib(type=JsonDict)
+    device_unused_fallback_keys = attr.ib(type=List[str])
```

Comment:
Should this (and the other variables etc. here) be called `device_unused_fallback_key_types`? It doesn't actually have the keys, correct?

Reply:
Yup.  I've updated the MSC and the js-sdk PR too